src/analysis/normed_space/star/continuous_functional_calculus.lean

```diff
@@ -0,0 +1,232 @@
+/-
+Copyright (c) 2022 Jireh Loreaux. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jireh Loreaux
+-/
+
+import analysis.normed_space.star.gelfand_duality
+import topology.algebra.star_subalgebra
+
+/-! # Continuous functional calculus
+
+In this file we construct the `continuous_functional_calculus` for normal elements of a (unital)
+C⋆-algebra over `ℂ`. This is a star algebra equivalence
+`C(spectrum ℂ a, ℂ) ≃⋆ₐ[ℂ] elemental_star_algebra ℂ a` which sends the (restriction of) the
+identity map `continuous_map.id ℂ` to the (unique) preimage of `a` under the coercion of
+`elemental_star_algebra ℂ a` to `A`.
+
+Being a star algebra equivalence between C⋆-algebras, this map is continuous (even an isometry),
+and by the Stone-Weierstrass theorem it is the unique star algebra equivalence which extends the
+polynomial functional calculus (i.e., `polynomial.aeval`).
+
```

Comment:
I'd like a sentence explaining this notion for people who are less familiar with it. Something like: 
```lean
For any continuous function `f : spectrum ℂ a →  ℂ`, this makes it possible to define an element `f a` in the original algebra, which heuristically has the same eigenspaces as `f` and acts on eigenvector of `f` for an eigenvalue `λ` as multiplication by `f λ`. This description is perfectly accurate in finite dimension, but only heuristic in infinite dimension as there might be no genuine eigenvector. In particular, when `f` is a polynomial `∑ cᵢ Xⁱ`, then `f a` is `∑ cᵢ aⁱ`. Also, `id a = a`.
```